### ISSUE TITLE
Uninstalled, now can't reinstall :-(

### ISSUE THREAD

**Issue by Jon2001** [GoogleChromeLabs/text-app#593]
I uninstalled it, decided that was a mistake, and now can't reinstall it.  When I first tried a month or two ago the message on the app page (as best as I remember) was that it was only available on Chromebooks (I am on a chromebook (Acer R11)).  Tonight I tried again and now it says it is "temporarily Unavailable".  Is there another way I can reinstall this app?

Thanks,
Jon

**Comment by stephengale801** [comment]
@Jon2001 Hoping this works. You should be able to install from the Chrome Webstore on a Chrome device.
https://chromewebstore.google.com/detail/text/mmfbcljfglbokpmkimbfghdkjmjhdgbg

**Comment by Jon2001** [comment]
@stephengale801 , Sorry for my bad manners.  Since writing the original message above I purchased a new chromebook and reset the old one.  That restored the Text app (or I may have reinstalled, but one of those two did it).  Between those two events I had guessed the original problem was that the old chromebook was no longer getting updates and maybe not running a current release was giving the webstore some pause.  Thank you very much for the link though, and again, sorry for missing your reply.  - Jon
